src/main/java/Duke/Task.java

```diff
@@ -0,0 +1,23 @@
+package Duke;
+
+public class Task {
+
+        protected String description;
+        protected boolean isDone;
+
+        public Task(String description) {
```

Comment:
you could create setters and getter functions as part of encapsulation? (i think)